adoc/chapters/programming_interface.adoc

```diff
@@ -21703,7 +21703,7 @@ reported by [code]#Group::fence_scope#).
 SYCL provides an algorithms library based on the functions described in Section
 28 of the {cpp17} specification.
 The first argument to each function is a <<group>>, and data ranges can be
-described using pointers, iterators or instances of the [code]#multi_ptr# class.
+described using pointers or instances of the [code]#multi_ptr# class.
```

Comment:
Maybe this is a separate PR, but the constraints on the functions below don't make it clear that `multi_ptr` is allowed.  They just say "`Ptr` is a pointer".

Reply:
I'd prefer to do it in a separate PR, if you don't mind.  It probably makes sense to fix the `Ptr` constraints while addressing #340, because we're already going to have to clarify what they can point _to_.

Reply:
Addressed as part of #730.